CryptoPkg/Driver/Packaging/generate_cryptodriver.py

```diff
@@ -825,7 +825,7 @@ class options():
         comp_types = get_supported_library_types(phase)
         upper_phase = phase.upper()
         for arch in arches:
-            if phase == "StandaloneMm" and arch != "X64":
+            if phase == "StandaloneMm" and (arch != "AARCH64" and arch != "X64"):

```

Comment:
should this be an or between AARCH64 and X64?

Reply:
I simplified it to remove negative logic.